pkg/common/common.go

```diff
@@ -7,6 +7,12 @@ import (
 
 var (
 	DEBUG         = os.Getenv("DEBUG")
+	SQL_DEBUG     = envToBool("SQL_DEBUG", false)
+	DB_TYPE       = getEnv("DB_TYPE", "sqlite3")
+	DB_HOST       = os.Getenv("DB_HOST")
+	DB_NAME       = os.Getenv("DB_NAME")
+	DB_USER       = os.Getenv("DB_USER")
+	DB_PASSWORD   = os.Getenv("DB_PASSWORD")
```

Comment:
maybe it would be better to have single `DATABASE_URL` var and then parse it for internal use?

Reply:
Done